src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -64,6 +63,7 @@ private void registerRubber() {
             registerNonPlaceableItem(Items.RUBBER, RubberSynthesizer.RECIPE_TYPE, SlimefunItems.OIL_BUCKET);
             new RubberSynthesizer().register(LiteXpansion.getInstance());
         }
+        rubberItem = SlimefunItem.getByID("RUBBER").getItem();
```

Comment:
This variable is only used in the one method, I don't see why this is even global. Just make it a final variable in the needed method.